integration_tests/src/main/python/hive_delimited_text_test.py

```diff
@@ -422,6 +422,31 @@ def create_hive_table_with_custom_timestamp_format(spark, property_location):
         conf=hive_text_enabled_conf)
 
 
+def test_read_compressed_hive_text(spark_tmp_table_factory):
+    """
+    This tests whether compressed Hive Text tables are readable from spark-rapids.
+    For GZIP compressed tables, spark-rapids should not attempt to split the input files.
+    """
+
+    table_name = spark_tmp_table_factory.get()
+
+    def create_table_with_compressed_files(spark):
+        spark.range(100000)\
+            .selectExpr("id",
+                        "cast(id as string) id_string")\
+            .repartition(1).write.format("hive").saveAsTable(table_name)
+
+    # Create Hive Text table with compression enabled.
+    with_cpu_session(create_table_with_compressed_files,
+                     {"hive.exec.compress.output": "true"})
```

Comment:
gzip is guaranteed here?  or should we set compression codec explicitly?  If it supports multiple, do we have tests for them?

Reply:
That's a good question. I've only seen GZIP/deflate. 

Let me check if I can swap the codec.

Reply:
I've added tests for DEFLATE, Gzip and Bzip2. There are other conceivable codecs, but I think this sample covers the reasonable gamut.